project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/SettingValueDataFixture.php

```diff
@@ -17,40 +19,93 @@ class SettingValueDataFixture extends AbstractReferenceFixture implements Depend
+            $cookies = $this->getReferenceForDomain(ArticleDataFixture::ARTICLE_COOKIES, $domainId);
+            $this->setting->setForDomain(Setting::COOKIES_ARTICLE_ID, $cookies->getId(), $domainId);
+
+            $personalDataDisplaySiteContent = t('By entering an email below, you can view your personal information that we register in our online store.
+                An email with a link will be sent to you after entering your email address, to verify your identity.
+                Clicking on the link will take you to a page listing all the personal details we have connected to your email address.', [], 'dataFixtures', $locale);
+            $this->setting->setForDomain(Setting::PERSONAL_DATA_DISPLAY_SITE_CONTENT, $personalDataDisplaySiteContent, $domainId);
+
+            $personalDataExportSiteContent = t('By entering an email below, you can download your personal and other information (for example, order history)
+                from our online store. An email with a link will be sent to you after entering your email address, to verify your identity.
+                Clicking on the link will take you to a page where you’ll be able to download these informations in readable format - it will be the data
+                registered to given email address on this online store domain.', [], 'dataFixtures', $locale);
+            $this->setting->setForDomain(Setting::PERSONAL_DATA_EXPORT_SITE_CONTENT, $personalDataExportSiteContent, $domainId);
+
+            $orderSentText = t('
+                <p>
+                    Order number {number} has been sent, thank you for your purchase.
+                    We will contact you about next order status. <br /><br />
+                    <a href="{order_detail_url}">Track</a> the status of your order. <br />
+                    {transport_instructions} <br />
+                    {payment_instructions} <br />
+                </p>
+            ', [], 'dataFixtures', $locale);
+            $this->setting->setForDomain(Setting::ORDER_SENT_PAGE_CONTENT, $orderSentText, $domainId);
+
+            /** @var \Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroup $pricingGroup */
+            $pricingGroup = $this->getReferenceForDomain(PricingGroupDataFixture::PRICING_GROUP_ORDINARY, $domainId);
+            $this->setting->setForDomain(Setting::DEFAULT_PRICING_GROUP, $pricingGroup->getId(), $domainId);
+
+            $this->setting->setForDomain(
+                SeoSettingFacade::SEO_META_DESCRIPTION_MAIN_PAGE,
+                t('Shopsys Framework - the best solution for your eshop.', [], 'dataFixtures', $locale),
+                $domainId
+            );
+            $this->setting->setForDomain(SeoSettingFacade::SEO_TITLE_MAIN_PAGE, t('Shopsys Framework - Title page', [], 'dataFixtures', $locale), $domainId);
+            $this->setting->setForDomain(SeoSettingFacade::SEO_TITLE_ADD_ON, t('| Demo eshop', [], 'dataFixtures', $locale), $domainId);
```

Comment:
Just referencing previous comment (https://github.com/shopsys/shopsys/pull/1425#discussion_r332065790) about better changing of setting values

Reply:
And I still disagree :smile:

Reply:
I still see it as a valid and almost painless refactoring which will make it more reaadable.